plugins/pulp_python/plugins/models.py

```diff
@@ -3,23 +3,58 @@
 import re
 import tarfile
 
+from mongoengine import StringField
+from pulp.server.db.model import FileContentUnit
+
 from pulp_python.common import constants
 
 
 DEFAULT_CHECKSUM_TYPE = 'sha512'
 
 
-class Package(object):
+# These are required to be in the PKG-INFO file.
+REQUIRED_ATTRS = ('name', 'version', 'summary', 'home_page', 'author', 'author_email', 'license',
+                  'description', 'platform')
+
+
+class Package(FileContentUnit):
     """
     This class represents a Python package.
```

Comment:
I like the idea, but agree that it's not needed for this PR. @bmbouter want to file a general issue about this for all Pulp models? We can do that when you become the documentation czar!